File: app/src/main/java/com/sygic/example/hello3dwiw/SygicNaviFragment.java
package com.sygic.example.hello3dwiw;

import com.sygic.aura.embedded.SygicFragmentSupportV4;

public class SygicNaviFragment extends SygicFragmentSupportV4 {

    @Override
    public void onResume() {
        startNavi();
        setCallback(new SygicNaviCallback(getActivity()));
        super.onResume();
    }
}

File: app/src/main/java/com/sygic/example/hello3dwiw/SygicNaviFragment.kt
package com.sygic.example.hello3dwiw

import com.sygic.aura.embedded.SygicFragmentSupportV4

class SygicNaviFragment : SygicFragmentSupportV4() {
    override fun onResume() {
        startNavi()
        setCallback(SygicNaviCallback(requireActivity()))
        super.onResume()
    }
}